### DISCUSSION TITLE
How to add IFrames to quasar page ?

### DISCUSSION THREAD

**Discussion by amirsafaricg** [quasarframework/quasar#10651] [General - Components / Directives / etc] [answered]
Hello everyone 
I dropped an html file into the src folder inside components and im trying to access it through an I Frame in a q-page 
but I get this error what does it mean ? 

`runtime-dom.esm-bundler.js?8886:10 GET http://localhost:8080/components/index.html 404 (Not Found)`

In browser it shows 
`Cannot GET /components/index.html`

how can i fix it ?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@amirsafaricg 

>  html file into the src folder inside components and im trying to access it through an I Frame

Could you explain what you want to achieve? I you just want to `display` the contents of a html file inside a q-page then there are other(better) ways to do that:
https://vuejs.org/v2/guide/syntax.html#Raw-HTML

If you really want to use an IFrame then there are some Vue specific IFrame libs:
https://github.com/officert/vue-friendly-iframe
https://github.com/chantouchsek/vue-iframes

**Comment by pikil** [comment]
@amirsafaricg I assume, there are some pre-rendered HTML files you are trying to use. In that case, you need to place them into ```public/templates/index.html``` folder instead and provide the link ```/templates/index.html``` for the iframe element.


* However, if that is possible, I would suggest to convert that html into ```.vue``` and lazy-import it (```component: () => import('components/path-to-vue-file')```) or use v-html="directive".
